android/assets/jsons/Civ V - Vanilla/Units.json

```diff
@@ -254,6 +270,22 @@
 		"uniques": ["+[50]% Strength vs [Mounted]","+10 HP when healing"],
 		"attackSound": "metalhit"
 	},
+	{
+		"name": "Battering Ram",
+		"replaces": "Spearman",
+		"uniqueTo": "Huns",
+		"unitType": "Melee",
+		"movement": 2,
+		"strength": 10,
+		"cost": 75,
+		"requiredTech": "Bronze Working",
+		"obsoleteTech": "Physics",
+		"upgradesTo": "Trebuchet",
+		"uniques": ["+[300]% Strength vs [City]", "No defensive terrain bonus", "+[-33]% Strength when defending",
```

Comment:
See, "+[-33]% Strength when defending" is why we need uniques that have the + inside the square brackets ;)

Reply:
Jup, I've been convinced. Changing all the uniques again is going to make a lot of people angry though.... I'll starting doing this in the future anyway

Reply:
... or from another json: `"+[-33]% Production when constructing [Military] units",` ...

Reply:
Since technically none of the old uniques have been deprecated, I think it's legitimate to make the changes to the 'even newer' and then deprecate the older ones.